src/it/java/org/checkstyle/base/AbstractItModuleTestSupport.java

```diff
@@ -131,54 +196,54 @@ protected final Checker createChecker(Configuration moduleConfig)
-                                    ModuleCreationOption moduleCreationOption)
+                                 ModuleCreationOption moduleCreationOption)
             throws Exception {
-        final Configuration dc;
+        final Checker checker = new Checker();
+        checker.setModuleClassLoader(Thread.currentThread().getContextClassLoader());
+        // make sure the tests always run with English error messages
+        // so the tests don't fail in supported locales like German
+        final Locale locale = Locale.ENGLISH;
+        checker.setLocaleCountry(locale.getCountry());
+        checker.setLocaleLanguage(locale.getLanguage());
 
         if (moduleCreationOption == ModuleCreationOption.IN_TREEWALKER) {
-            dc = createTreeWalkerConfig(moduleConfig);
+            final Configuration dc = createTreeWalkerConfig(moduleConfig);
+            checker.configure(dc);
         }
         else if (ROOT_MODULE_NAME.equals(moduleConfig.getName())) {
-            dc = moduleConfig;
+            checker.configure(moduleConfig);
         }
         else {
-            dc = createRootConfig(moduleConfig);
+            final Configuration dc = createRootConfig(moduleConfig);
+            checker.configure(dc);
         }
-
-        final Checker checker = new Checker();
-        // make sure the tests always run with English error messages
-        // so the tests don't fail in supported locales like German
-        final Locale locale = Locale.ENGLISH;
-        checker.setLocaleCountry(locale.getCountry());
-        checker.setLocaleLanguage(locale.getLanguage());
-        checker.setModuleClassLoader(Thread.currentThread().getContextClassLoader());
-        checker.configure(dc);
         checker.addListener(getBriefUtLogger());
         return checker;
     }
 
     /**
-     * Creates {@link DefaultConfiguration} or the {@link Checker}.
-     * based on the given {@link Configuration}.
+     * Creates {@link DefaultConfiguration} for the {@link TreeWalker}
+     * based on the given {@link Configuration} instance.
      *
      * @param config {@link Configuration} instance.
-     * @return {@link DefaultConfiguration} for the {@link Checker}.
+     * @return {@link DefaultConfiguration} for the {@link TreeWalker}
+     *     based on the given {@link Configuration} instance.
      */
-    protected final DefaultConfiguration createTreeWalkerConfig(Configuration config) {
+    protected static DefaultConfiguration createTreeWalkerConfig(Configuration config) {
         final DefaultConfiguration dc =
-                new DefaultConfiguration("configuration");
+                new DefaultConfiguration(ROOT_MODULE_NAME);
         final DefaultConfiguration twConf = createModuleConfig(TreeWalker.class);
         // make sure that the tests always run with this charset
-        dc.addProperty("charset", "iso-8859-1");
+        dc.addProperty("charset", StandardCharsets.UTF_8.name());
```

Comment:
These two charsets are not equivalent, why did we change this? `StandardCharsets` has a constant for this charset.

Reply:
I see this more as probably fixing a mistake. Both test areas have the same comment, similar code but each one has its own "charset". I am syncing the code between the 2 areas and do not see why the charsets would be different since they are both basically doing the same thing.

This has been here since 2015. The commit was 437986510709d7abc64b59c72779343ac5e77599 and I could not find the PR to see what documentable reason for this charset.

https://checkstyle.org/styleguides/google-java-style-20180523/javaguide.html#s2.2-file-encoding
Google's guide specifically says UTF8, so it makes even weirder we are using a different charset for our tests than they recommend.

Reply:
Yeah, I can’t imagine that this would cause any issues. It’s weird that we used two different charsets for basically the same thing. 

Resolved.